src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -42,8 +51,10 @@ export default function VAOSBreadcrumbs({ children, labelOverride }) {
         label: 'My HealtheVet',
       },
       {
-        href: manifest.rootUrl,
+        // For Breadcrumb objects that are router links, the href is relative to the manifest.rootUrl
+        href: '/',
         label: 'Appointments',
+        isRouterLink: true,
```

Comment:
We can keep the href as the `manifest.rootUrl` so that it appears as that URL when the user hovers or inspects the element. And in L21 function above, we can test for the `href === manifest.rootUrl` and if so, push to `/` instead of the URL.